cli/tests/test_cli.py

```
@@ -0,0 +1,1491 @@
+                
+                # Mock matrix definition response with mismatched labels
+                mock_get.return_value.status_code = 200
+                mock_get.return_value.json.return_value = {
+                    "json_definition": json.dumps(mismatched_matrix)
+                }
+                
+                # Mock risk assessment creation
+                mock_post.return_value.status_code = 201
+                mock_post.return_value.json.return_value = {"id": 100}
+                
+                result = runner.invoke(import_risk_assessment, [
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)